src/WordPress/DataStore.php

```diff
@@ -59,6 +60,18 @@ public function getHostAPIUserUniqueId()
     }
 
     /**
+     * @return bool
+     */
+    public function getClientAPICredentialsExist()
```

Comment:
I might rename this to `clientAPICredentialsExist()`. "get" implies it returns something.